File: Assets/Scripts/Runes/AddTag.cs
using UnityEngine;
using System.Collections;

public class AddTag : Rune {

	public string minion_uid { get; set; }

	public string tag { get; set; }

	public override void Execute (System.Action action)
	{
		action ();
	}

	public override void OnGUI ()
	{
		
	}
}

File: Server/runes/AddTag.js
var entities = require('../entityManager');
var Rune = require('../RuneVM')



exports.execute = function (rune, state) {
     var ent = entities.getEntity(rune["target"], state);
     ent.tags[rune.tag] = true;
}

exports.CreateRune = function (sourceGuid, targetGuid, tag)
{
    var rune = {
        "runeType":"AddTag",
        "source":sourceGuid,
        "target":targetGuid,
        "tag":tag
    }
    return rune;
}

exports.canSee = function (rune, controller, state) {
    return true;
}